### ISSUE TITLE
Tasks/config.js: Adopt RWE UK approach

### ISSUE THREAD

**Issue by kettanaito** [actum/gulp-dev-stack#62]
## 1. Structure

RWE UK has a really good approach towards declaring file paths in the project. Basically, it looks like this:

``` js
const
    DEVELOPMENT_BASE     = '.',
    BUILD_BASE           = './build',
    ...
    CSS_BASE             = `${DEVELOPMENT_BASE}/scss`,
    CSS_ENTRY            = `${CSS_BASE}/main.scss`,
    CSS_ALL              = `${CSS_BASE}/scss/**/*.scss`,
    CSS_BUILD            = `${BUILD_BASE}/css`,
    ....
```

Zoltan and I both think this is an approach to follow, since this way we do not re-declare the paths, but rather use previously declared constants (which is impossible in default `module.exports` structure, as it exports the Object).
## 2. Singular import/export

Another good point is to export each of the declared paths separately. This will turn the following code:

``` js
const config = require('./config.js');
const paths = config.paths;
const development = paths.development;
...
gulp.src(development.css.all)
    .pipe(...);
```

into something much more readable and maintainable:

``` js
const config = require('./config.js');
...
gulp.src(config.CSS_ALL)
    .pipe(...);
```

This may be achieved by exporting each path separately:

``` js
/* config.js */
const CSS_ALL = `${CSS_BASE}/scss/**/*.scss`;
...
module.exports = {
    CSS_ALL
};
```

What are you thinking about it?

**Comment by kettanaito** [comment]
Implemented:
https://github.com/actum/gulp-dev-stack/commit/71556f38261b049b6331097c7cfb98b038f01364
https://github.com/actum/gulp-dev-stack/commit/6d5428be060749f38b8af59c7d70894388e36532 (Changes according to @janpanschab comment).

If you have any suggestions how to make this even better, feel free to comment on this issue. Thank you.

**Comment by janpanschab** [comment]
@kettanaito Are we able to merge this into master? Could you create PR?

**Comment by kettanaito** [comment]
@janpanschab I need to fix some linting on it. This was the reason to move it all at once, because now we use new ESLint, but do not have the proper changes on `config-new`, which were established in another branch.

**Comment by janpanschab** [comment]
closed by #81
